starfish/util/click.py

```diff
@@ -0,0 +1,47 @@
+import click
```

Comment:
idea: could we import all of click into this module, and then replace the option symbol?

i.e.,
```
from click import *

click_option = option
def option(*args, **kwargs):
    kwargs["cls"] = RequiredParentOption
    return click_option(*args, **kwargs)
```

And then where we currently import click, we would just do:

`from starfish.util import click`

and then the rest of the code should basically be unmodified.

Reply:
I don't see any idiom for doing `import *`; I'll import explicitly and push.

Reply:
Pushed.